src/views/List.jsx

```diff
@@ -6,13 +6,9 @@ export function List({ data }) {
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
-			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
-			</ul>
+			{data.map((data) => (
+				<ul>{ListItem(data)}</ul>
```

Comment:
`ul` is the whole list so it should be outside of the map function. And keep only the ListItem. in addition. We need to use the ListItem as a component and not as a function. Finally you need to check what exists inside the ListItem to see if you need to modify smth in there